tests/test_bayes_factor.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import numpy as np
+import arviz as az
+
+from src.arviz_stats.bayes_factor import bayes_factor
+
+def test_bayes_factor_comparison():
+    idata = az.from_dict(
+        posterior={"a": np.random.normal(1, 0.5, 5000)},
+        prior={"a": np.random.normal(0, 1, 5000)}
+    )
```

Comment:
```suggestion
    idata = from_dict({"posterior": {"a": np.random.normal(1, 0.5, (2, 1000))},
                        "prior": {"a": np.random.normal(0, 1, (2, 1000))}})
```

change all other call to `from_dict` following this pattern